apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -23,16 +33,47 @@ const AnimationPickerBody = React.createClass({
 
   getInitialState() {
     return {
-      searchQuery: ''
+      searchQuery: '',
+      categoryQuery: ''
     };
   },
 
   onSearchQueryChange(value) {
     this.setState({searchQuery: value});
   },
 
+  onCategoryChange(category) {
+    this.setState({categoryQuery: category});
+  },
```

Comment:
One thing you could do with these is name your param the same as your state field, then take advantage of an ES6 feature, where `{ foo }` is the same thing as `{ foo: foo }`. Up do you which approach you prefer.
```
onCategoryChange(categoryQuery) {
  this.setState({categoryQuery});
}
```